testing/install_external_dependencies.sh

```diff
@@ -44,6 +44,6 @@ if [[ -n "$PYTHON" ]] ; then
   conda update -q conda
   conda info -a
   conda config --add channels conda-forge
-  conda install -q matplotlib pandasql ipython jupyter_client ipykernel matplotlib bokeh
+  conda install -q matplotlib pandasql ipython jupyter_client ipykernel matplotlib bokeh=0.12.6
```

Comment:
since this is from the other commit, I think it's better if you rebase your PR

Reply:
It's @zjffdu 's hot fix, I rebase my branch for ci check. you can ignore it

Reply:
@felixcheung  you can focus on the change of Paragraph.java